src/app/pages/quickMaps/components/sideNavContent/sideNavContent.component.scss

```diff
@@ -160,4 +176,42 @@ h3 {
 
 ::ng-deep .mat-select-panel.allow-long-text {
   max-width: unset;
+}
+
+mat-slide-toggle {
+  position: absolute;
+  left: 0em;
+  top: 0em;
+}
+
+::ng-deep {
+
+  & .mat-tooltip {
+    font-size: 12px;
+  }
+
+  & .mat-slide-toggle-label {
+    padding: 1em;
+  }
+
+  & .mat-slide-toggle-thumb {
+    position: relative !important;
+    background: $color_blue_form !important;
+    border-radius: 3px !important;
+    width: 20px !important;
+    height: 15px !important;
+    margin-top: 3px;
+  }
+
+  & .mat-slide-toggle-thumb:before {
```

Comment:
You could also nest this inside the above block and just use:
&:before {

Reply:
doesn't seem to like that :/ not sure why it wouldn't work though the more I think about it